src/Core/src/Core/HandlerMauiAppBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Hosting.Internal;
+
+namespace Microsoft.Maui
+{
+	public static class HandlerMauiAppBuilderExtensions
```

Comment:
I am thinking that since handlers are a core function of Maui, this should be a property on the AppBuilder? I know I just made a comment saying we need rules, but I think this is so fundamental that an app can't run without it.